pp/integrations/plugin-packs/procedures/operatingsystems-mac-snmp.md

```diff
@@ -2,48 +2,853 @@
+| Disk-IO       | OS-Mac-Disk-IO-SNMP-custom | Check disk access. For each check the name of the disk will appear as a "label" rather than the letter assigned                             |
+| Load          | OS-Mac-Load-SNMP-custom    | Check the server load average                                                                                                                      |
+| Memory        | OS-Mac-Memory-SNMP-custom  | Check the rate of memory usage |
+| Swap          | OS-Mac-Swap-SNMP-custom    | Check virtual memory usage                                                                                                                         |
+
+> The services listed above are created automatically when the **OS-Mac-SNMP-custom** host template is used.
+
+</TabItem>
+<TabItem value="Not attached to a host template" label="Not attached to a host template">
+
+| Service Alias        | Service Template                        | Service Description                                                                               | Discovery  |
+|:---------------------|:----------------------------------------|:--------------------------------------------------------------------------------------------------|:----------:|
+| Disk-Generic-Id      | OS-Mac-Disk-Generic-Id-SNMP-custom      | Check the rate of free space on the disk. For each check the name of the disk will appear        |            |
+| Disk-Generic-Name    | OS-Mac-Disk-Generic-Name-SNMP-custom    | Check the rate of free space on the disk. For each checks the mount pont of the disk will appear  | X          |
+| Disk-Global          | OS-Mac-Disk-Global-SNMP-custom          | Check the rate of free space on disks. For each checks the mount point of disks will appear       |            |
+| Inodes-Global        | OS-Mac-Inodes-Global-SNMP-custom        | Check Inodes space usage on partitions                                                            |            |
+| Process-Generic      | OS-Mac-Process-Generic-SNMP-custom      | Check Unix process/service is working                                                             |            |
+| Traffic-Generic-Id   | OS-Mac-Traffic-Generic-Id-SNMP-custom   | Check the bandwidth of the interface. For each checks the name of the interface will appear       |            |
+| Traffic-Generic-Name | OS-Mac-Traffic-Generic-Name-SNMP-custom | Check the bandwidth of the interface. For each checks the name of the interface will appear       | X          |
+| Traffic-Global       | OS-Mac-Traffic-Global-SNMP-custom       | Check the bandwidth of interfaces. For each checks the name of the interface will appear          |            |
+| Uptime               | OS-Mac-Uptime-SNMP-custom               | Time since the server has been working and available                                              |            |
```

Comment:
```suggestion
| Disk-Generic-Id      | OS-Mac-Disk-Generic-Id-SNMP-custom      | Check the rate of free space on the disk. For each check the name of the disk will appear        |            |
| Disk-Generic-Name    | OS-Mac-Disk-Generic-Name-SNMP-custom    | Check the rate of free space on the disk. For each checks the mount pont of the disk will appear  |            |
| Disk-Global          | OS-Mac-Disk-Global-SNMP-custom          | Check the rate of free space on disks. For each checks the mount point of disks will appear       | X          |
| Inodes-Global        | OS-Mac-Inodes-Global-SNMP-custom        | Check Inodes space usage on partitions                                                            |            |
| Process-Generic      | OS-Mac-Process-Generic-SNMP-custom      | Check Unix process/service is working                                                             |            |
| Traffic-Generic-Id   | OS-Mac-Traffic-Generic-Id-SNMP-custom   | Check the bandwidth of the interface. For each checks the name of the interface will appear       |            |
| Traffic-Generic-Name | OS-Mac-Traffic-Generic-Name-SNMP-custom | Check the bandwidth of the interface. For each checks the name of the interface will appear       |            |
| Traffic-Global       | OS-Mac-Traffic-Global-SNMP-custom       | Check the bandwidth of interfaces. For each checks the name of the interface will appear          |  X         |
| Uptime               | OS-Mac-Uptime-SNMP-custom               | Time since the server has been working and available                                              |            |
```